src/styles/animations.css

```
@@ -13,12 +19,63 @@
+   AUTO-ANIMATE PATTERN (Modern @starting-style)
+   ============================================ */
+/* Automatically animates children when they appear/disappear with display: none */
+
+.auto-animate > * {
+  /* Modern browsers with @starting-style support */
+  @starting-style {
+    opacity: 0;
+    transform: translateX(10px);
+  }
+
+  opacity: 1;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

opacity is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)